spec/models/post_mover_spec.rb

```diff
@@ -314,6 +319,18 @@
 
       end
 
+      context "moving from a deleted topic" do
+        before do
+          topic.trash!
+        end
+
+        it "adds a moderator post at the location of the first moved post" do
+          topic.move_posts(user, [p2.id, p4.id], title: "new testing topic name")
+
+          expect(topic.posts).to include(have_attributes(post_type: Post.types[:small_action], action_code: "split_topic", post_number: 2, sort_order: 2))
```

Comment:
Makes sense to me and matches better the test description. What about 825089bf5b675d496032e479a4c0a2935c94889b?